<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sureDialog</class>
 <widget class="QDialog" name="sureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Are you sure?</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="imageLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="sureButton">
       <property name="text">
        <string>Sure(%1)(F11)</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/sure.png</normaloff>:/icons/sure.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="retryButton">
       <property name="text">
        <string>Retry(F12)</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/retry.png</normaloff>:/icons/retry.png</iconset>
       </property>
       <property name="shortcut">
        <string>F12</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
